packages/api/tests/lambdas/test-sf-semaphore-down.js

```diff
@@ -170,6 +210,33 @@ test('sfSemaphoreDown lambda throws error for invalid event message', async (t)
   );
 });
 
+test('sfSemaphoreDown lambda decrements semaphore for s3-stored event message', async (t) => {
+  const status = 'SUCCEEDED';
+  const { client, semaphore } = t.context;
+  const queueName = randomId('low');
+  // arbitrarily set semaphore so it can be decremented
+  await client.put({
+    TableName: process.env.SemaphoresTable,
+    Item: {
+      key: queueName,
+      semvalue: 1
+    }
+  }).promise();
+
+  const cloudwatchConfig = { status, queueName };
+  const stubReturn = packagedEventReturn(queueName);
+  const pullStepFunctionStub = sinon.stub(commonAws, 'pullStepFunctionEvent');
+  const proxiedFunction = proxyquire('../../lambdas/sf-semaphore-down', { pullStepFunctionEvent: pullStepFunctionStub }).handleSemaphoreDecrementTask;
+  pullStepFunctionStub.returns(stubReturn);
```

Comment:
I guess since no other tests in this file are going to go down this code path, it's fine to not run this test in serial or restore the stub?

Reply:
Hmm, no, let's fix that.

Reply:
380a681